runtests

```diff
@@ -45,6 +46,12 @@ runtest() {
   set -o pipefail
   start=$(date +%s.%3N)
 
+  if [[ -z "$subfolder" ]]; then
```

Comment:
Better to put this code block before the line `start=$(daate +%s.%3N)` to record a more accurate start time for calculating test duration.

Reply:
Yep, make sense.

Will improve it.

Thanks!